src/LinAlg/hiopMatrixSparseTripletStorage.hpp

```diff
@@ -134,11 +134,11 @@ namespace hiop {
 
   protected:
     friend class hiopMatrixComplexSparseTriplet;
-  
+
     Tidx nrows, ncols, nnz;
-  
+
     Tidx *irow, *jcol;
-    Tval *values; 
+    Tval *values;
```

Comment:
```suggestion
    Tval* values_;
```
Perhaps it would be good to follow HiOp style guidelines.